[2024-03-21 18:16:01,666] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T18:15:00+00:00 [queued]>
[2024-03-21 18:16:01,684] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T18:15:00+00:00 [queued]>
[2024-03-21 18:16:01,684] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 18:16:01,684] {taskinstance.py:1088} INFO - Starting attempt 1 of 6
[2024-03-21 18:16:01,685] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 18:16:01,695] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2024-03-21T18:15:00+00:00
[2024-03-21 18:16:01,702] {standard_task_runner.py:52} INFO - Started process 33450 to run task
[2024-03-21 18:16:01,705] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', '2024-03-21T18:15:00+00:00', '--job-id', '10135', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpbjvyxiv6', '--error-file', '/tmp/tmpvetp679q']
[2024-03-21 18:16:01,705] {standard_task_runner.py:77} INFO - Job 10135: Subtask retrieve_raw_data
[2024-03-21 18:16:01,749] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.retrieve_raw_data 2024-03-21T18:15:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 18:16:01,811] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=retrieve_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T18:15:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T18:15:00+00:00
[2024-03-21 18:16:03,528] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 18:16:03,541] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20240321T181500, start_date=20240321T181601, end_date=20240321T181603
[2024-03-21 18:16:03,584] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-03-21 18:16:03,607] {local_task_job.py:151} INFO - Task exited with return code 0
